### ISSUE TITLE
Sentence construction for translations

### ISSUE THREAD

**Issue by rickj** [w3c/publ-a11y#460]
https://github.com/w3c/publ-a11y/blob/main/a11y-meta-display-guide/2.0/draft/localizations/en-US/display_guide_vocabulary_w3c_en-US.json provides clear sentences for the different conditions for translation, except for the "detailed-Conformance" section.

It appears this is constructed to break out different variables that can be assembled to create a sentence that reflects the claim, however, this is only true for English.  It is impossible to construct an accurate set of logic to construct translations of the various parts that would make sense when assembled together in a similar way for every language.

I would like to recommend that instead we construct all possible sentences, so translations can be possible, and logic to select the right one remains predictable.

**Comment by GeorgeKerscher** [comment]
I can see how the suggested approach would make it possible to do the translations.

It looks like this change would need to be made quickly to get into the canonicle JSON file.

**Comment by GeorgeKerscher** [comment]
In determing  the number of strings we will need if we eliminate the concatanated array , I have calculated it as:
the formulae is n(n+1)/2 +1   and then one more for  not known)

So the numberof variables are listed below for the sections:

Navigation: 4= 11 strings plus not known
hazards 3=7 strings plus not known and not present
Additional information 5 = 16 + not present.
Clarity 7=29 + not known.

Any thoughts on how to approach this?

**Comment by jonaslil** [comment]
I'm adding some thoughts on the Conformance section.

I don't see an issue with concatenating strings like " EPUB Accessibility 1.1 ",  " WCAG 2.1 ", " Level AA". 

The problem arises because the instructions try to present all information as full sentences, which doesn't seem necessary.

Steps 19-23 (ONIX) and 18-22 (EPUB) are especially problematic. These steps construct a sententence by concatenating small pieces: "The publication was certified " + " on " + certification_date + " by " + certifier + " with a credential of " +  certifier_credentials. (The instructions don't explicitly say that these should be concatenated; the reader must infer it.)

Instead, we could present the information as a list or group of separate, simple explanations:

- Certified by: [certifier]
- Credential: [certifier_credentials]
- Certification date: [certification_date]

This avoids concatenation of strings that need localization.

Another way to handle localization of sentences or phrases with differing structures in different languages would be to use placeholder variables within the phrases that are localized. But explaining how this would work is beyond my competence.

Side note:
The intended display outcome of this last step is not quite clear to me:
"For more information refer to the certifier's report" with certifier_report as a link.

**Comment by GeorgeKerscher** [comment]
We have removed the concatination of statements. Now they are a list of single items
